include/tue/config/write.h

```diff
@@ -11,6 +11,7 @@ namespace config
 
 enum WriteType
 {
+    XML,
```

Comment:
I don't think the XML writer was implemented. Do we need it?

Reply:
Keep it here and throw an error